src/react/utils/useProxy.ts

```diff
@@ -19,10 +19,14 @@ import { useSnapshot } from 'valtio/react'
  * }
  *
  * @param proxy
+ * @param options
  * @returns A new proxy which you can use in the render as well as in callbacks.
  */
-export function useProxy<T extends object>(proxy: T): T {
-  const snapshot = useSnapshot(proxy) as T
+export function useProxy<T extends object>(
+  proxy: T,
+  options?: Parameters<typeof useSnapshot>[1]
```

Comment:
To be precise, let's do this:
```suggestion
  options?: NonNullable<Parameters<typeof useSnapshot>[1]>
```

Reply:
I don't mind going safe here ofc 👍🏿  (but since I wanna learn possible edge-cases and got not much experience in OSS (unlike you :P) I ask: Is `null` even a possible arg value here? When I f.e. try to assign null as Parameter it doesn't work)

Reply:
It's to avoid `undefined`. There's no runtime risk to pass `undefined`. It's just to make the `useProxy` type same as `useSnapshot` type.